sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -236,6 +236,8 @@ object ScalaNativePluginInternal {
       val cpppaths  = (cwd ** "*.cpp").get.map(_.abs)
       val paths     = cpaths ++ cpppaths
 
+      logger.info(s"Using $gc gc")
```

Comment:
I'm not completely sold on printing this in every project that uses Scala Native. It's useful for developing Scala Native but should not be shown by default.

Reply:
Perhaps it should be a debug level log message instead. (sorry, for the intrusion in your PR)

Reply:
This was mainly used as a debug tool so I could guarantee that each Scala Native link was using the correct setting so debug probably make more sense. I wasn't completely sold but it does tell what is happening. Maybe "Using" could be " Linking" or something as well?